extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/AggregateOptions.java

```diff
@@ -114,6 +115,17 @@ public AggregateOptions hint(Bson hint) {
         return this;
     }
 
+    /**
+     * Sets the batchSize.
```

Comment:
Can you improve the JavaDoc to explain how this is used ?
You can check the JavaDoc of `com.mongodb.reactivestreams.client.AggregatePublisher.batchSize(int)` as it's the method being called with this attribute.

Reply:
I don't know if the second part is necessary, but anyways, I copied it.